content/en/docs/tasks/configure-pod-container/migrate-from-psp.md

```diff
@@ -19,30 +19,552 @@ admission controller. This can be done effectively using a combination of dry-ru
+appropriate Pod Security profile is applied to new namespaces.
+
+You can also statically configure the Pod Security admission controller to set a default enforce,
+audit, and/or warn level for unlabeled namespaces. See [Configure the Admission
+Controller](docs/tasks/configure-pod-container/enforce-standards-admission-controller/#configure-the-admission-controller)
+for more information.
+
+## 5. Disable PodSecurityPolicy
+
+Finally, you're ready to disable PodSecurityPolicy. To do so, you will need to modify the admission
+configuration of the API server: [How do I turn off an admission
+controller?](/docs/reference/access-authn-authz/admission-controllers/#how-do-i-turn-off-an-admission-controller).
+
+To verify that the PodSecurityPolicy admission controller is no longer enabled, you can manually run
+a test by impersonating a user without access to any PodSecurityPolicies (see the [PodSecurityPolicy
+example](/docs/concepts/policy/pod-security-policy/#example)), or by verifying in the API server
+logs. At startup, the API server outputs log lines listing the loaded admission controller plugins:
+
+```
+I0218 00:59:44.903329      13 plugins.go:158] Loaded 16 mutating admission controller(s) successfully in the following order: NamespaceLifecycle,LimitRanger,ServiceAccount,NodeRestriction,TaintNodesByCondition,Priority,DefaultTolerationSeconds,ExtendedResourceToleration,PersistentVolumeLabel,DefaultStorageClass,StorageObjectInUseProtection,RuntimeClass,DefaultIngressClass,MutatingAdmissionWebhook.
+I0218 00:59:44.903350      13 plugins.go:161] Loaded 14 validating admission controller(s) successfully in the following order: LimitRanger,ServiceAccount,PodSecurity,Priority,PersistentVolumeClaimResize,RuntimeClass,CertificateApproval,CertificateSigning,CertificateSubjectRestriction,DenyServiceExternalIPs,ValidatingAdmissionWebhook,ResourceQuota.
+```
+
+You should see `PodSecurity` (in the validating admission controllers), and neither list should
+contain `PodSecurityPolicy`.
+
+Once you are certain the PSP admission controller is disabled (and after sufficient soak time to be
+confident you won't need to roll back), you are free to delete your PodSecurityPolicies and any
+associated Roles, ClusterRoles, RoleBindings and ClusterRoleBindings (just make sure they don't
+grant any other unrelated permissions).
+
+## Appendix: Pod Security Policies to Pod Security Standards {#psp-fields}
+
+The following table enumerates the fields in the `PodSecurityPolicySpec` API, along with the allowed
+values for the [Baseline](/docs/concepts/security/pod-security-standards/#baseline) and
+[Restricted](/docs/concepts/security/pod-security-standards/#restricted) profiles. Anything outside
+the allowed values for those profiles would fall under the
+[Privileged](/docs/concepts/security/pod-security-standards/#priveleged) profile.
+
+<table>
+	<caption style="display:none"></caption>
```

Comment:
For accessibility, we should set a caption.

Reply:
Oops, thanks.